basedtyping/__init__.py

```diff
@@ -347,3 +347,13 @@ def issubform(
     if forminfo is Never:
         return False
     return issubclass(form, forminfo)  # type: ignore[arg-type]
+
+
+# `_SpecialForm`s init isn't typed
+@_SpecialForm  # type: ignore[call-arg]
+def Untyped(self: _SpecialForm, parameters: object) -> NoReturn:
+    """Special type indicating that something isn't typed.
+
+    This is more specialized than ``Any`` and can help with gradually typing modules.
+    """
+    raise TypeError(f"{self} is not subscriptable")
```

Comment:
what does this actually do? wouldnt it be simpler to just do this?
```suggestion
Untyped = Any
"""Special type indicating that something isn't typed.

This is more specialized than ``Any`` and can help with gradually typing modules.
"""
```

Reply:
If it's separate then it would be discernable at runtime.